api_tests/nodes/views/test_node_list.py

```diff
@@ -740,7 +737,7 @@ def test_create_component_inherit_contributors(self):
     def test_create_component_inherit_contributors_with_unregistered_contributor(self):
         parent_project = ProjectFactory(creator=self.user_one)
         parent_project.add_unregistered_contributor(
-            fullname='far', email='bar', permissions=[permissions.READ],
+            fullname='far', email='foo@bar', permissions=[permissions.READ],
```

Comment:
Ah, that's right. It's still ambiguous whether a valid email is expected, maybe do `foo@bar.baz`?